experimental/rsocket/RSocketRequestHandler.h

```diff
@@ -31,6 +32,29 @@ class RSocketRequestHandler {
   virtual yarpl::Reference<yarpl::flowable::Flowable<reactivesocket::Payload>>
   handleRequestStream(
       reactivesocket::Payload request,
-      reactivesocket::StreamId streamId) = 0;
+      reactivesocket::StreamId streamId) {
+    return yarpl::flowable::Flowables::error<reactivesocket::Payload>(
+        std::make_exception_ptr(
```

Comment:
You can just pass in the `std::logic_error` directly.  There's an `Flowables::error<T, ExnType>` overload.